mpi4jax/_src/xla_bridge/__init__.py

```diff
@@ -34,6 +34,10 @@ def _is_truthy(str_val):
 
```

Comment:
The CPU client still requires these names to be bytes? Or could we transition to strings throughout?

Reply:
I was trying to not change things that do not need changing.

This was a bug in the plugin interface on top of which only one version of the cuda backend is built.
In the future it will support both.

Other backends, I believe, already sanitise the inputs and support both.

Reply:
Alright, your call. I don't see much of a reason to keep things as bytes if all backends support str, but whatever you think is easiest.

Reply:
ok then, I'll move everything to normal strings

Reply:
that is supposed to be less brittle anyway, because that's the one that jax internally uses..